frontend/src/pages/GroupPage.jsx

```diff
@@ -1,15 +1,159 @@
 import Template from "../components/template";
+import ModalHeader from "../components/modalHeader.jsx";
+import ModalLeftWindow from "../components/modalLeftWindow";
+import Button from "../components/button";
+import { useState, React, useEffect } from "react";
+import AddMemberWindow from "../components/addMemberWindow";
+import "../css/personalPage.css";
+import UploadNotesWindow from "../components/uploadNotesWindow";
+import { useParams } from "react-router";
 
 function GroupPage(props) {
+    const [buttonAddMember, setButtonAddMember] = useState(false);
+    const [groupProfile, setButtonGroupProfile] = useState(false);
```

Comment:
For consistency, this should be ``buttonGroupProfile``

Reply:
Changed it.